### PR TITLE
[IMP] account: rename Refund to Credit Note

### PR DESCRIPTION
**Pull request by dbh-odoo**
Task: https://www.odoo.com/web?#id=32014&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
Pad: https://pad.odoo.com/p/r.6e7b31c5954af7b3cef7d0dea8237eae

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/models/account_invoice.py [hunk 1]
```diff
@@ -461,7 +461,7 @@ def compute_taxes(self):
     def unlink(self):
         for invoice in self:
             if invoice.state not in ('draft', 'cancel'):
-                raise UserError(_('You cannot delete an invoice which is not draft or cancelled. You should refund it instead.'))
+                raise UserError(_('You cannot delete an invoice which is not draft or cancelled. You should create Credit Note instead.'))
```
Comment (bst-odoo) [inline addons/account/models/account_invoice.py]: You cannot delete an invoice which is not draft or cancelled. You should create a credit note instead.

File: addons/account/views/account_invoice_view.xml [hunk 2]
```diff
@@ -219,7 +219,7 @@
                 <header>
                         <button name="action_invoice_open" type="object" states="draft" string="Validate" class="oe_highlight" groups="account.group_account_invoice"/>
                         <button name="%(action_account_invoice_payment)d" type="action" states="open" string="Register Payment" groups="account.group_account_invoice" class="oe_highlight"/>
-                        <button name="%(action_account_invoice_refund)d" type='action' string='Ask Refund' groups="account.group_account_invoice" attrs="{'invisible': ['|',('type', 'in', ['in_refund','out_refund']),('state','not in',('open','paid'))]}"/>
+                        <button name="%(action_account_invoice_refund)d" type='action' string='Add Credit Note' groups="account.group_account_invoice" attrs="{'invisible': ['|',('type', 'in', ['in_refund','out_refund']),('state','not in',('open','paid'))]}"/>
```
Comment (bst-odoo) [inline addons/account/views/account_invoice_view.xml]: Ask for a credit note

File: addons/gamification_sale_crm/data/gamification_sale_crm_data.xml [hunk 3]
```diff
@@ -102,8 +102,8 @@
             <field name="domain">[('state','!=','cancel'),('user_id','=',user.id),('type','=','out_refund')]</field>
         </record>
         <record model="gamification.goal.definition" id="definition_crm_tot_customer_refunds">
-            <field name="name">Total Customer Refunds</field>
-            <field name="description">The total refunded value is a negative value. Validated when higher (min refunded).</field>
+            <field name="name">Total Customer Credit Notes</field>
+            <field name="description">The value of total credit note is a negative. Validated when higher (min credit note value).</field>
```
Comment (bst-odoo) [inline addons/gamification_sale_crm/data/gamification_sale_crm_data.xml]: The total credit note value is negative. Validated when higher (min credit note value).

**Comment by danger89** [comment]
Is is not already call refund current version (v11) ? Maybe pull the latest changes in.. Or close?

**Pull request closed without merging**